assets/js/blocks/cart/test/block.js

```diff
@@ -32,6 +32,11 @@ import OrderSummarySubtotalBlock from '../inner-blocks/cart-order-summary-subtot
 import OrderSummaryShippingBlock from '../inner-blocks/cart-order-summary-shipping/frontend';
 import OrderSummaryTaxesBlock from '../inner-blocks/cart-order-summary-taxes/frontend';
 
+jest.mock( '@wordpress/compose', () => ( {
+	...jest.requireActual( '@wordpress/compose' ),
+	useResizeObserver: jest.fn().mockReturnValue( [ null, { width: 0 } ] ),
```

Comment:
As mentioned in my commit this is something I have actually run into before, if you're including useResizeObserver as a dependency outside of WP (such as in a jest test) then you get an error because the underlying library (react-resize-aware) cannot access the jsx runtime due to how it's built. See https://github.com/FezVrasta/react-resize-aware/issues/58

If we had pnpm we could force resolution to this newer version, but in npm the solutions to this are hackier so we just stub it out here instead.